qa/tasks/rgw.py

```diff
@@ -221,6 +218,8 @@ def ship_apache_configs(ctx, config, role_endpoints, on_client = None,
     finally:
         log.info('Removing apache config...')
         for client in clients_to_create_as:
+            cluster_name, daemon_type, client_id = teuthology.split_role(client)
+            client_with_cluster = '.'.join((cluster_name, daemon_type, client_id))
```

Comment:
I wanted the cluster name and client id.
I see from the logs the client id is already logged and I can find out what is cluster name from the teuthology node. 
No need to add more logging